src/ConfirmDeleteAllModal.js

```diff
@@ -18,43 +18,31 @@ type ConfirmDeleteAllModalProps = {
 class ConfirmDeleteAllModal extends React.Component<ConfirmDeleteAllModalProps, {}> {
     render() {
         return (
-            <Modal
-                aria-labelledby='deleteAll-button'
+            <ModalWithFooter
                 show={this.props.show}
-                onHide={this.props.onCancel}
-                size='lg'
-                dialogClassName='ConfirmDeleteAllModal'
-                centered>
-                <Modal.Body className='ConfirmDeleteAllModal__content'>
-                    <div className='ConfirmDeleteAllModal__header'>
-                        <span role='img' aria-hidden='true' >
-                            <ErrorIcon className='ConfirmDeleteAllModal__warning-svg' />
-                        </span>
-                        <FormattedMessage id='ConfirmDeleteAllModal.title' />
-                    </div>
-                    <div className='ConfirmDeleteAllModal__footer'>
-                        <Button
-                            className='ConfirmDeleteAllModal__option-button cancel'
-                            onClick={this.props.onCancel}>
-                            <FormattedMessage id='ConfirmDeleteAllModal.cancelButton' />
-                        </Button>
-                        <Button
-                            id='deleteAll-button'
-                            className='ConfirmDeleteAllModal__option-button confirm'
-                            onClick={this.props.onConfirm}>
-                            <FormattedMessage id='ConfirmDeleteAllModal.confirmButton' />
-                        </Button>
-                    </div>
-                </Modal.Body>
-            </Modal>
+                focusOnOpenSelector={'#DeleteAll-done'}
+                focusOnCloseSelector={'.ProgramBlockEditor__program-deleteAll-button'}
+                ariaLabel={this.props.intl.formatMessage({id: 'ConfirmDeleteAllModal.confirmButton'})}
+                onClose={this.props.onCancel}
+                buttonProperties={
+                    [
+                        {label: this.props.intl.formatMessage({id: 'ConfirmDeleteAllModal.cancelButton'}), onClick: this.props.onCancel},
+                        {id: 'DeleteAll-done', label: this.props.intl.formatMessage({id: 'ConfirmDeleteAllModal.confirmButton'}), onClick: this.props.onConfirm, isPrimary: true}
```

Comment:
Let's use the full component name as the id prefix, and "confirm" rather than "done" to match our other naming: `ConformDeleteAllModal-confirm`